clubManagement/templates/clubManagement/statusreport_list.html

```diff
@@ -34,13 +34,23 @@ <h1 id="page-header">Status updates</h1>
                             </tbody>
                         </table>
                     </div>
+                    <button onclick="addStatusUpdate()" class="col-md-2 offset-md-10 col-sm-6 btn btn-primary save-btn">Add
+                    </button>
                 </div>
             </div>
         </div>
     </div>
     {% include 'base/footer.html' %}
 {% endblock %}
 
+{%  block script %}
+    <script>
+        function addStatusUpdate() {
+            window.location = '/club/status/create';
```

Comment:
Don't hard code the links. Use Django url patterns {% url 'status_create' %} in an a tag.